app/code/Magento/CustomerGraphQl/Model/Resolver/GenerateCustomerToken.php

```diff
@@ -54,7 +68,11 @@ public function resolve(
 
         try {
             $token = $this->customerTokenService->createCustomerAccessToken($args['email'], $args['password']);
-            return ['token' => $token];
+            $tokenLifetime = $this->scopeConfig->getValue(self::TOKEN_LIFETIME_PATH_KEY);
+            return [
+                'token' => $token,
+                'expiration_time' => is_numeric($tokenLifetime) && $tokenLifetime > 0 ? $tokenLifetime : 1
```

Comment:
Is it better to use `ttl` (time to live) instead of `expiration_time`, I think `expiration_time` is used for date time or timestamp value, but in our case this is just number of hours

Reply:
@codynguyen1 it totally makes sense. Let's rename it to `ttl` then. Thank you!